src/protocol/boolean/bitwise_equal.rs

```diff
@@ -46,6 +46,26 @@ where
     no_ones(ctx, record_id, &xored_bits).await
 }
 
+pub async fn bitwise_equal_gf2<C, S>(
+    ctx: C,
+    record_id: RecordId,
+    a: &[S],
+    b: &[S],
+) -> Result<S, Error>
+where
+    C: Context,
+    S: LinearSecretSharing<Gf2> + BasicProtocols<C, Gf2>,
+{
+    debug_assert!(a.len() == b.len());
+    let c = a
+        .iter()
+        .zip(b.iter())
```

Comment:
```suggestion
    let c = zip(a.iter(), b.iter())
```
Might be easier to read.